src/create-jobs.js

```diff
@@ -78,10 +90,11 @@ const createPHPUnitJobs = function (config) {
  * @return {Job}
  */
 const createPHPUnitJob = function (version, deps, config) {
+    const fileName = appendDistExtensionIfFilenameDoesntExist('phpunit.xml');
```

Comment:
Change the name of the constant to "configFileName" or "phpunitConfigFile"; "fileName" is to ambiguous.